person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(name, age = 'Unknown', permission: true)
```

Comment:
```suggestion
  def initialize(name, age = 'Unknown', permission: true)
  @id = Random.rand(1..1000)
```

- [ ] Kindly add `id` in your initialized method it should be  a random integer between 1 and 1000 👍🏿

Reply:
thanks for the advice, I will ad the `id` for every object created